lib/install/examples/react/babel.config.js

```diff
@@ -18,15 +18,17 @@ module.exports = function(api) {
   return {
     presets: [
       isTestEnv && [
-        require('@babel/preset-env').default,
+        '@babel/preset-env',
         {
           targets: {
             node: 'current'
-          }
-        }
+          },
+          modules: 'commonjs'
+        },
+        '@babel/preset-react'
```

Comment:
hi may I know what is this for? why is the '@babel/preset-react' mentioned 2 times and the first one is in 1 array with @babel/preset-env ? thanks